lib/libspl/include/sys/time.h

```diff
@@ -58,6 +58,11 @@
 #define	NSEC2MSEC(n)    ((n) / (NANOSEC / MILLISEC))
 #endif
 
+typedef long long       hrtime_t;
+typedef struct timespec timestruc_t;
+typedef struct timespec timespec_t;
+
```

Comment:
@behlendorf ive moved the typedefs as you suggested in previous commit, and i have changed longlong_t to long long as longlong_t is not defined by any standard so musl does not support it.